spec.md

```diff
@@ -0,0 +1,315 @@
+
+- **`mediaType`** *string*  
+  This REQUIRED property uniquely identifies the artifact for computer consumption. It may be owned by an org or a company and MUST be globally unique and versioned.  
+  The format of `mediaType` MUST use the following format:  
+  `application/vnd.`[org|company]`.`[objectType]`.`[optionalSubType]`.config.`[version]`+json`
+- **`title`** *string-string map*  
+  This REQUIRED property must have at least one value, representing the name of the type displayed for human consumption. The title may be displayed in a repository listing, or registry tooling.  
+  Title is a collection of localized strings, indexed with [ISO Language Codes][iso-lang-codes].
+  - **`locale`** *string*  
+    2-2 language codes representing the country and locale. [ISO Language Codes][iso-lang-codes]
+  - **`title`** *string*  
+Localized title. The max length MUST not exceed 30 characters and MUST not encode formatting characters.
+- **`description`** *string-string map*  
+    This REQUIRED property must have at least one value, providing a short description of the type for human consumption. The description may be displayed in repository listings or registry tooling.  
+    Description is a collection of localized strings, indexed with [ISO Language Codes][iso-lang-codes].  
+  - **`locale`** *string*  
+    2-2 language codes representing the country and locale. [ISO Language Codes][iso-lang-codes]
+  - **`description`** *string*  
+Localized description. The max length MUST not exceed 255 characters and MUST not encode formatting characters.
+- **`moreInfoUrl`** *url*  
+This OPTIONAL property provides additional details, intended for consumers of the type. This is most often a marketing & awareness overview page.
+- **`specUrl`** *url*  
+This OPTIONAL property references a spec, providing additional details on the type.
+- **`tools`** *string-string map*  
+  This OPTIONAL property provides a collection of tools that may be used with artifact type. The property is intended for end users to find more info on how to find and install related tools. Registry operators MAY provide links to the tools in their repository listing.
+  - **`url`** *url*  
+    This REQUIRED property links to a page where users can download the tool. The URL MAY be a direct link to a download URL, or a link to documentation for how to download the tool.
+  - **`title`** *string-string map*  
+    This OPTIONAL property represents the name of the tool, displayed for human consumption. The title may be displayed in a repository listing, or registry tooling.  
+    Title is a collection of localized strings, indexed with [ISO Language Codes][iso-lang-codes].
+    - **`locale`** *string*  
+      2-2 language codes representing the country and locale. [ISO Language Codes][iso-lang-codes]
+    - **`title`** *string*  
+Localized title. The max length MUST not exceed 30 characters and MUST not encode formatting characters.
+- **`configSchemaReferenceUrl`** *url*  
```

Comment:
We should indicate that this is JSON Schema, which version, and link to the JSON schema spec

Reply:
This was one of the recent conversations that we decided to add other formats. For instance, someone might want to use yaml instead of json. I've added details to **Defining a Unique Artifact Type** that discusses the different formats. And, if they use standard text formats, registry operators have a better likelyhood to parse them. That said, parsing a config object is optional. I suspect most wont create config objects.

Reply:
Sorry, I meant we should require JSON schema for the *schema* reference, not for the config itself, but the point is taken... I wonder if JSON schema is extensible enough to define  the schema for YAML as well (when transformed)